client/ayon_maya/plugins/publish/collect_farm_instances.py

```diff
@@ -0,0 +1,31 @@
+import pyblish.api
+from ayon_maya.api import plugin
+
+
+class CollectFarmInstances(plugin.MayaInstancePlugin):
+    """Collect Farm Instances for remote publish
+    """
+
+    order = pyblish.api.CollectorOrder + 0.223
+    label = "Collect Farm Instances"
+    families = ["animation", "pointcache"]
```

Comment:
Are we not supporting other product types, like `usd`, `fbx`, yeti caches and potentially ornatrix caches? Since it fully triggers logic 'locally' on the farm machine it should be trivial to support most of the regular publish product types, right? (because it behaves as if local but on the farm machine?)

Reply:
Actually - no... I was stupid on my comment there. On the farm job it'd still run that full extractor.. ignore me.